js/private/repository_utils.bzl

```diff
@@ -0,0 +1,25 @@
+"""
+Repository rule utilities.
+"""
+
+def _config_settings_impl(rctx):
+    v = native.bazel_version.split(".")
+    is_6_3_or_greater = len(v) > 1 and v[0].isdigit() and (int(v[0]) < 6 or (int(v[0]) == 6 and v[1].isdigit() and int(v[1]) < 3))
```

Comment:
This was originally using bazel_skylib [versions.bzl](https://github.com/bazelbuild/bazel-skylib/blob/288731ef9f7f688932bd50e704a91a45ec185f9b/lib/versions.bzl) however we can't even use that dep because it is loaded within the `rules_js_dependencies()` which is too late because this file is also loaded there.

The unit tests only passed because bazel_skylib is in the dev repos for this repository, but e2e tests were failing when using bazel_skylib.